lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -34,6 +36,15 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
         <h2 class="mb-4 uppercase font-semibold text-gray-500">
           Running sessions (<%= length(@sessions) %>)
         </h2>
+        <div class="-mt-6 text-md text-gray-500 font-medium">
+          <% memory = memory_info(@sessions) %>
```

Comment:
We don't want use variables in HEEx templates, because this disables some of the smart change tracking that LiveView does. Instead, you can compute the memory in `mount` and then on every `{:session_change, _}`, assign it and use directly in the template :)